### PR TITLE
Create despero.ms

### PR DESCRIPTION
**Pull request by Boofurd**

### CODE REVIEW

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                "/run 0-9",
+                "/haste 0-9",
+                "/flyspeed <number>",
+                "/fly",
+                "/speak <player> <message>",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/disguiseviewself",
+                "/confuse <player>",
+                "/tp <player>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: there shouldn't be a comma on this line

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
+                "/flyspeed <number>",
+                "/fly",
+                "/speak <player> <message>",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/disguiseviewself",
+                "/confuse <player>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF mind control

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                "/dismountplayer",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
+                "/flyspeed <number>",
+                "/fly",
+                "/speak <player> <message>",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/disguiseviewself",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF disguising as another being

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
+                "/flyspeed <number>",
+                "/fly",
+                "/speak <player> <message>",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF controlling anothers mind

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            'op': 'false',
+            'speech': "I will destroy the Justice League for what they have done to me!",
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
+                "/flyspeed <number>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF flying at incredible speeds

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+            'trail': '',
+            'op': 'false',
+            'speech': "I will destroy the Justice League for what they have done to me!",
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
+                "/run 0-9",
+                "/haste 0-9",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF superstrength equal or exceeding that of superman

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODUxMTcwMTYyMTIsInByb2ZpbGVJZCI6IjI1N2E0NWQ2N2RiMDQ5MmQ5NDVjYTAzMGY0OTg1OTMyIiwicHJvZmlsZU5hbWUiOiJCb29mdXJkIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9lNzZlZGM1YzEzMmRlZTE1YjVmNTI4NDFjMDcxZTYxMjZhMzVmYzc4ZTlkODNjNmJlMzI5NTYxNDU2ZTM3ODI0In19fQ==',
+            'trail': '',
+            'op': 'false',
+            'speech': "I will destroy the Justice League for what they have done to me!",
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
+                "/run 0-9",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF running as fast as the flash

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODUxMTcwMTYyMTIsInByb2ZpbGVJZCI6IjI1N2E0NWQ2N2RiMDQ5MmQ5NDVjYTAzMGY0OTg1OTMyIiwicHJvZmlsZU5hbWUiOiJCb29mdXJkIiwic2lnbmF0dXJlUmVxdWlyZWQiOnRydWUsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS9lNzZlZGM1YzEzMmRlZTE1YjVmNTI4NDFjMDcxZTYxMjZhMzVmYzc4ZTlkODNjNmJlMzI5NTYxNDU2ZTM3ODI0In19fQ==',
+            'trail': '',
+            'op': 'false',
+            'speech': "I will destroy the Justice League for what they have done to me!",
+            'powers': array(
+                "/buff",
+                "/carryplayer",
+                "/dismountplayer",
+                "/jump 0-9",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF jumping as high as onepunchman

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+                "/fly",
+                "/speak <player> <message>",
+                "/mindcontrol <playername>",
+                "/stopmindcontrol",
+                "/disguiseviewself",
+                "/confuse <player>",
+                "/tp <player>",
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 24 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF strength exceeding that of superman

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+                "/effect ". @plr ." 10 9999 2 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF very fast regen

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+                "/effect ". @plr ." 11 9999 3 true",
+                "/effect ". @plr ." 13 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF him being able to breathe underwater or in space

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+                "/effect ". @plr ." 16 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF being able to see in the dark

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+                "/effect ". @plr ." 21 9999 24 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF him exceeding superman's durability

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+                "/give ". @plr ." minecraft:iron_boots 1 8:30 4:30 name:&4S&ep&4e&ce&ed&8s&0t&9e&1r_&4Boots",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF him equaling flash/speedster speeds

File: CommandHelper/LocalPackages/commands/buffs/despero.ms [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+                "/shot give ". @plr ." energybeam"
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: VID OR PICTURE REQUIRED OF having a laser beam

File: CommandHelper/LocalPackages/commands/buffs/despero.ms (hunk 9 above)
Comment (Boofurd) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: https://www.bing.com/images/search?q=despero+vs+justice+league&view=detailv2&&id=0CFC0240AB7FC249239B659992129B79E44ABE25&selectedIndex=23&ccid=4To3disp&simid=607990104232428802&thid=OIP.Me13a37762b29c3a36da37c050829790ao0&ajaxhist=0

File: CommandHelper/LocalPackages/commands/buffs/despero.ms (hunk 13 above)
Comment (Boofurd) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: https://www.bing.com/images/search?q=despero+vs+superman&view=detailv2&adlt=strict&id=FD2B528B51B95A40381ABEA748D993810F272D67&selectedIndex=25&ccid=nc2U9td6&simid=608031314449270049&thid=OIP.M9dcd94f6d77a11738f942982f259eafeo0&ajaxhist=0

Easily takes supergirl's hit, and blasts laser

File: CommandHelper/LocalPackages/commands/buffs/despero.ms (hunk 15 above)
Comment (Boofurd) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: https://www.bing.com/images/search?q=despero+vs+superman&view=detailv2&adlt=strict&id=FD2B528B51B95A40381ABEA748D993810F272D67&selectedIndex=25&ccid=nc2U9td6&simid=608031314449270049&thid=OIP.M9dcd94f6d77a11738f942982f259eafeo0&ajaxhist=0

shoots laser at supergirl

File: CommandHelper/LocalPackages/commands/buffs/despero.ms (hunk 11 above)
Comment (Boofurd) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: https://www.bing.com/images/search?q=despero+in+space&view=detailv2&&id=59136CCF886F3250F74EA10C85D7CAA9CB8C3E03&selectedIndex=8&ccid=6jx%2b4WAF&simid=608030077505701430&thid=OIP.Mea3c7ee160053fb5c2ba6c55d19ab1ebo0&ajaxhist=0

flying through space

File: CommandHelper/LocalPackages/commands/buffs/despero.ms (hunk 3 above)
Comment (Boofurd) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: I thought I'd leave this in so they could see their skin, will remove it

File: CommandHelper/LocalPackages/commands/buffs/despero.ms (hunk 5 above)
Comment (Boofurd) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: https://www.bing.com/images/search?q=despero+in+space&view=detailv2&&id=59136CCF886F3250F74EA10C85D7CAA9CB8C3E03&selectedIndex=8&ccid=6jx%2b4WAF&simid=608030077505701430&thid=OIP.Mea3c7ee160053fb5c2ba6c55d19ab1ebo0&ajaxhist=0

flies through space VERY quickly

File: CommandHelper/LocalPackages/commands/buffs/despero.ms (hunk 7 above)
Comment (Boofurd) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: This was my mistake, will remove

File: CommandHelper/LocalPackages/commands/buffs/despero.ms (hunk 15 above)
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/despero.ms]: sorry thats a fanfic that comes from pinterest

**Pull request merged**